package.json

```diff
@@ -23,7 +24,6 @@
     "webpack": "^2.6.1"
   },
   "dependencies": {
-    "react": "^15.6.0",
-    "react-dom": "^15.6.0"
+    "react-lite": "0.15.30"
```

Comment:
After some discussion on irc we came to the conclusion that this is "locked down" already